--- vendor/phpspec/prophecy/src/Prophecy/Call/CallCenter.php	2020-09-01 00:00:00.000000000 +0900
+++ vendor/phpspec/prophecy/src/Prophecy/Call/CallCenter.php	2020-09-01 00:00:00.000000000 +0900
@@ -176,7 +176,7 @@
 
         $expected = array();
 
-        foreach (call_user_func_array('array_merge', $prophecy->getMethodProphecies()) as $methodProphecy) {
+        foreach (call_user_func_array('array_merge', array_values($prophecy->getMethodProphecies())) as $methodProphecy) {
             $expected[] = sprintf(
                 "  - %s(\n" .
                 "%s\n" .
