--- vendor//phpunit/phpunit/src/Util/Annotation/DocBlock.php	2020-09-07 18:36:30.000000000 +0900
+++ vendor//phpunit/phpunit/src/Util/Annotation/DocBlock.php	2020-09-07 18:42:30.000000000 +0900
@@ -497,12 +497,12 @@
         if ($reflector instanceof \ReflectionClass) {
             $annotations = \array_merge(
                 $annotations,
-                ...\array_map(
+                ...\array_values(\array_map(
                     function (\ReflectionClass $trait): array {
                         return self::parseDocBlock((string) $trait->getDocComment());
                     },
                     \array_values($reflector->getTraits())
-                )
+                ))
             );
         }
 
